packages/adminrouter/extra/src/README.md

```diff
@@ -870,3 +1112,842 @@ expressions/statements:
 This is why the context of each endpoint is not protected by locks, in
 case when it's only about fetching a single value from context dict or
 storing/appending one there.
+
+## Tests writing howto
+
+This section is intended to quickly enable contributors to start writing tests.
```

Comment:
Instead:

"This section intends to enable contributors to write tests."

Let's not judge how quickly this is possible :-).